examples/window/window/app.py

```diff
@@ -167,6 +167,20 @@ def startup(self):
             "Change content", on_press=self.do_next_content, style=btn_style
         )
         btn_hide = toga.Button("Hide", on_press=self.do_hide, style=btn_style)
+        screen_change_btns_box = toga.Box(
+            children=[toga.Label(text="Move current window to:")]
+        )
+
+        def do_screen_change(screen):
+            self.current_window.screen = screen
+
+        for screen in self.screens:
+            screen_change_btns_box.add(
+                toga.Button(
+                    text=screen.name,
```

Comment:
A minor cosmetic suggestion: it's easier to reason about the screens if they're presented visually in the order they are on your desktop, annotated with which screen number they are.
```suggestion
        for index, screen in sorted(enumerate(self.screens), key=lambda s: s[1].origin):
            screen_change_btns_box.add(
                toga.Button(
                    text=f"{index}: {screen.name}",
```

With this patch, this presents the buttons as:
[2: External Monitor] [0: External monitor] [1: Built-in display]

whereas without it, the left hand button would move the screen to the middle display, the right hand button would move the window to the left hand display, and the centre button would move the window to the right hand display.

It doesn't make sense to do this at the core API level because Screen 0 (the primary, which anchors absolute coordinates) won't necessarily be the left most.